.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```

Comment:
If a repo would like to use their own template, they would be able to override it with a file named `bug_report.md` at the same location, right?

Reply:
yes exactly